src/OrleansCodeGenerator/GrainMethodInvokerGenerator.cs

```diff
@@ -265,7 +290,7 @@ private static MethodDeclarationSyntax GenerateInvokeMethod(Type grainType, Meth
             Type grainType,
             IdentifierNameSyntax grain,
             MethodInfo method,
-            IdentifierNameSyntax arguments)
+            ExpressionSyntax arguments)
```

Comment:
Could you explain about what this part of the change does please?

s/IdentifierNameSyntax arguments/ExpressionSyntax arguments/